---
title: Multivariate adaptive shrinkage improves cross-population transcriptome prediction
author: Daniel Araujo
date: '2023-02-28'
slug: multivariate-adaptive-shrinkage-improves-cross-population-transcriptome-prediction
categories:
  - Downloads
tags: []
---

Download data from: https://zenodo.org/record/7551845#.Y_5QTLTMIqs

Preprint from Wheeler lab sharing multi-ancestry prediction models from TOPMED/MESA  https://www.biorxiv.org/content/10.1101/2023.02.09.527747v1

## Title

Multivariate adaptive shrinkage improves cross-population transcriptome prediction for transcriptome-wide association studies in underrepresented populations

## Abstract

Transcriptome prediction models built on European-descent individuals' data are less accurate when applied to different populations because of differences in linkage disequilibrium patterns and allele frequencies. We hypothesized multivariate adaptive shrinkage may improve cross-population transcriptome prediction, as it leverages effect size estimates across different conditions - in this case, different populations. To test this hypothesis, we made transcriptome prediction models for use in transcriptome-wide association studies (TWAS) using different methods (Elastic Net, Matrix eQTL and Multivariate Adaptive Shrinkage in R (MASHR)) and tested their out-of-sample transcriptome prediction accuracy in population-matched and cross-population scenarios. Additionally, to evaluate model applicability in TWAS, we integrated publicly available multi-ethnic genome-wide association study (GWAS) summary statistics from the Population Architecture using Genomics and Epidemiology Study (PAGE) and Pan-UK Biobank with our developed transcriptome prediction models. In regard to transcriptome prediction accuracy, MASHR models had similar performance to other methods when the training population ancestry closely matched the test population, but outperformed other methods in cross-population predictions. Furthermore, in multi-ethnic TWAS, MASHR models yielded more discoveries that replicate in both PAGE and PanUKBB across all methods analyzed, including loci previously mapped in GWAS and new loci previously not found in GWAS. Overall, our study demonstrates the importance of using methods that benefit from different populationsâ€™ effect size estimates in order to improve TWAS for multi-ethnic or underrepresented populations.