---
title: 'GTEx v6 Expression models'
author: PredictDB Team
date: '2016-11-16'
slug: 'gtex-v6-expression-models'
categories:
  - Downloads
  - Models
tags: []
---
Expression prediction models with LD reference data are available in this [Zenodo repository](https://zenodo.org/record/3572842#.XfuyLNl7m90).

- [GTEx-V6p-1KG-2016-11-16.tar.gz](https://zenodo.org/record/3572842/files/GTEx-V6p-1KG-2016-11-16.tar.gz?download=1): Expression prediction models and LD reference, using variants defined in 1000 Genomes
- [GTEx-V6p-HapMap-2016-09-08.tar.gz](https://zenodo.org/record/3572842/files/GTEx-V6p-HapMap-2016-09-08.tar.gz?download=1): Expression prediction models and LD reference, using variants defined in Hapmap CEU track
- [snp_covariance_v6p.txt.gz](https://zenodo.org/record/3572842/files/snp_covariance_v6p.txt.gz?download=1): S-MultiXcan LD reference for HapMap models

Additional support information and details are available in the Zenodo repository. Please check [this link](http://hakyimlab.org/post/2017/v7-v6p-analysis/), about the GTEx v7 models, for a few caveats found in GTEx v6 models.

**Acknowledging these models**: If you use these models in your research, we ask you to cite:

- ["A gene-based association method for mapping traits using reference transcriptome data"](https://www.nature.com/articles/ng.3367), Gamazon et al, 2015, Nature Genetics
- ["Exploring the phenotypic consequences of tissue specific gene expression variation inferred from GWAS summary statistics"](https://www.nature.com/articles/s41467-018-03621-1), Barbeira et al, 2018, Nature Communications

If you use S-MultiXcan, we ask you to cite:

- ["Integrating predicted transcriptome from multiple tissues improves association detection"](https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1007889), Barbeira et al, 2019, PLOS Genetics